docs/en/02_Developer_Guides/00_Model/04_Data_Types_and_Casting.md

```diff
@@ -73,7 +73,13 @@ When adding a new `$db` field to a DataObject you can specify a default value
 to be applied to all existing records when the column is added in the database
 for the first time. This will also be applied to any newly created objects
 going forward. You do this be passing an argument for the default value in your 
-`$db` items. For example:
+`$db` items. 
+
+For integer values, the default is the first parameter in the field specification.
```

Comment:
I really feel we should remove this feature and make it all conform to using `$defaults`